avaframe/ana4Stats/getStats.py

```diff
@@ -82,35 +54,53 @@ def extractMaxValues(inputDir, cfgMain, avaDir, varPar, nameScenario='', paramet
     # load dataFrame of all simulation configurations
     simDF = cfgUtils.createConfigurationInfo(avaDir, standardCfg='')
 
-    # load peakFiles of all simulations and generate dictionary
+    # load peakFiles of all simulations and generate dataframe
     peakFilesDF = fU.makeSimDF(inputDir, avaDir=avaDir)
     nSims = len(peakFilesDF['simName'])
+    # initialize peakValues dictionary
     peakValues = {}
     for sName in simDF['simName'].tolist():
         peakValues[sName] = {}
 
-    # Loop through peakFiles and compute probability
+    # Loop through result field files and compute statistical measures
     for m in range(nSims):
-
+        # filter simulations according to parametersDict
         if peakFilesDF['simName'][m] in simNameList:
 
             # Load data
             fileName = peakFilesDF['files'][m]
             simName = peakFilesDF['simName'][m]
-            data = np.loadtxt(fileName, skiprows=6)
-
-            # compute max
-            max = np.amax(data)
-
+            dataFull = IOf.readRaster(fileName)
```

Comment:
read dataFull only in the else to avoid overwriting it in the if

Reply:
dataFull is required in both cases, it is the actual result field, inside the if the dataFull is restricted to where dataPFD > 0, inside the else it is not restricted

Reply:
true I did not see it...